spec/models/game_spec.rb

```diff
@@ -43,4 +43,46 @@
       expect(game.white_to_move).to eq(true)
     end
   end
+
+  describe "Populate Board" do
+    it "should have correct number of pieces" do
+      expect(game.pieces.count).to eq (32)
+    end
+
+    it "should be empty on the squares" do
+      expect(game.pieces.where(rank: 3..6)).to eq([])
+    end
+
+    (1..8).each do |file|
+      it "should have a white pawn at rank 2, file #{file}" do
+        pieces = game.pieces.where(rank: 2 , file: file)
+        expect(pieces.present?).to eq(true)
+        expect(pieces.count).to eq(1)
+        expect([pieces.first.type, pieces.first.color]).to eq(["Pawn", "white"])
+      end
+
+      it "should have a black pawn at rank 7, file #{file}" do
+        pieces = game.pieces.where(rank: 7 , file: file)
+        expect(pieces.present?).to eq(true)
+        expect(pieces.count).to eq(1)
```

Comment:
These two lines are redundant. Unlike `find_by`, which returns `nil` when it doesn't find anything, `where` will always return an `ActiveRecord` object that responds to #count, so it should be enough to just verify the count.